src/components/snapshot/ManagedSnapshotAccess.tsx

```diff
@@ -1,14 +1,16 @@
 import { Grid, Typography } from '@mui/material';
 import AddUserAccess, { AccessPermission } from 'components/common/AddUserAccess';
 import UserList from 'components/UserList';
-import React from 'react';
+import React, { Dispatch, useEffect } from 'react';
 import { getRoleMembersFromPolicies } from 'libs/utils';
 import _ from 'lodash';
-import { PolicyModel, SnapshotRequestModelPolicies } from 'generated/tdr';
+import { PolicyModel, SnapshotModel, SnapshotRequestModelPolicies } from 'generated/tdr';
 import { connect } from 'react-redux';
 import { TdrState } from 'reducers';
-import { SnapshotRoles } from 'constants';
 import { styled } from '@mui/material/styles';
+import { getDatasetPolicy } from 'actions';
+import { Action } from 'redux';
+import { DatasetRoles, SnapshotRoles } from 'src/constants';
```

Comment:
This import change was suggested by sonar and IntelliJ thinks it's better too. The code seems to run OK but maybe there's a reason why the code was importing from `'constants'` before?